examples/Atlas/+atlasUtil/plotWalkingTraj.m

```diff
@@ -58,7 +58,7 @@
   qd=xt(nq+(1:nq));
   qdd=qddtraj.eval(t);
 
-  kinsol = doKinematics(r,q);
+  kinsol = doKinematics(r, q, qd);
```

Comment:
Well really it was wrong ever since the creation (or move) of the file. See
this revision:
https://github.com/tkoolen/drake/blob/d6a2e5026387bf59dabf8734371eba59eb6ab141/examples/Atlas/%2BatlasUtil/plotWalkingTraj.m

It's calling doKinematics without qd, but then calling forwardJacDot after.

On Sat, Apr 11, 2015 at 10:04 PM, Hongkai Dai <notifications@github.com>
wrote:

> In examples/Atlas/+atlasUtil/plotWalkingTraj.m
> <https://github.com/RobotLocomotion/drake/pull/927#discussion_r28200060>:
>
> > @@ -58,7 +58,7 @@
> >    qd=xt(nq+(1:nq));
> >    qdd=qddtraj.eval(t);
> >
> > -  kinsol = doKinematics(r,q);
> > +  kinsol = doKinematics(r, q, qd);
>
> I made an error in this line in pull request #961
> <https://github.com/RobotLocomotion/drake/pull/961> , Twan fixed this for
> me since we are computing Jdotv later
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RobotLocomotion/drake/pull/927/files#r28200060>.
>